src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -11,12 +11,13 @@
 import com.minecolonies.coremod.colony.WorkOrderView;
 import com.minecolonies.coremod.colony.buildings.BuildingTownHall;
 import com.minecolonies.coremod.colony.permissions.Permissions;
+import com.minecolonies.coremod.configuration.Configurations;
 import com.minecolonies.coremod.lib.Constants;
-import com.minecolonies.coremod.network.messages.PermissionsMessage;
-import com.minecolonies.coremod.network.messages.RecallTownhallMessage;
-import com.minecolonies.coremod.network.messages.ToggleJobMessage;
-import com.minecolonies.coremod.network.messages.WorkOrderChangeMessage;
+import com.minecolonies.coremod.network.messages.*;
 import com.minecolonies.coremod.util.LanguageHandler;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.block.state.IBlockState'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)